gef-ui/server/controllers/application-details/index.js

```diff
@@ -249,9 +249,10 @@ const applicationDetails = async (req, res, next) => {
       params.errors = req.errors;
     }
 
-    if (req.success) {
-      params.success = req.success;
-    }
```

Comment:
Function `applicationDetails` is used for 5 routes. Maybe it is better to leave original code also, so we would have 2 sources for message: `req.success` and `req.flash('success')`.

Reply:
Added back original code